src/Crime/CrimeHelpers.ts

```diff
@@ -44,7 +44,7 @@ export function findCrime(roughName: string): Crime | null {
     return Crimes.DealDrugs;
   } else if (roughName.includes("bond") && roughName.includes("forge")) {
     return Crimes.BondForgery;
-  } else if ((roughName.includes("traffic") || (roughName.includes("illegal")) && roughName.includes("arms")) {
+  } else if ((roughName.includes("traffic") || roughName.includes("illegal")) && roughName.includes("arms")) {
```

Comment:
i think this is now fixed in dev

Reply:
yep thanks for the heads up, dropped the extra commit from this branch now